pkg/envoy/proxy_test.go

```diff
@@ -59,23 +83,27 @@ func TestProxy(t *testing.T) {
 		return true
 	}, 2*time.Second, 50*time.Millisecond)
 
+	// Check the output from fake-envoy to make sure we didn't hit an error case
+	require.Empty(t, envoyOut.String())
+	require.Empty(t, envoyErr.String())
```

Comment:
If the process hit an error case, such as a missing flag expected by `fake-envoy`, you would never know about in this test other than seeing the output file never being written.

With this change, we check for the output file with just an assert -- which allows the test to continue on -- and then make assertions about the output from Envoy so that errors become obvious when you're developing tests.